code/game/objects/items/devices/geiger_counter.dm

```diff
@@ -229,24 +243,22 @@
 
 
 /obj/item/geiger_counter/cyborg
-	var/mob/listeningTo
 
-/obj/item/geiger_counter/cyborg/equipped(mob/user)
-	. = ..()
-	if(listeningTo == user)
-		return
-	if(listeningTo)
-		UnregisterSignal(listeningTo, COMSIG_ATOM_RAD_ACT)
-	RegisterSignal(user, COMSIG_ATOM_RAD_ACT, PROC_REF(redirect_rad_act))
-	listeningTo = user
+/obj/item/geiger_counter/cyborg/activate_self(mob/user)
```

Comment:
You probably still want to do something on `equipped`, so that the counter resumes working if they pull the module back out, rather than needing to turn it off and on again.